### PR TITLE
add publisher_blocks_overridden to imp.deal.ext

### PR DESCRIPTION
**Pull request by yosei-ito**

### CODE REVIEW

File: version2.en.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Moreover, this specification does not contain description of general RTB protoco
     * [native Object](#native-object)
     * [pmp Object](#pmp-object)
     * [deal Object](#deal-object)
+    * [deal extension Object](#dealext-object)
```
Comment (yowcow) [inline version2.en.md]: `s/deal extension/deal.ext/` should make it look more consistent

File: version2.req-examples.md [hunk 2]
```diff
@@ -1059,7 +1059,9 @@
             "at": 1,
             "bidfloor": 4.050223,
             "bidfloorcur": "USD",
-            "ext": {},
+            "ext": {
+              "publisher_blocks_overridden": 1
+            },
```
Comment (yowcow) [inline version2.req-examples.md:1064]: 👍

**Review by yowcow — COMMENTED** [review]
added a comment pls check

File: version2.en.md (hunk 1 above)
Comment (yosei-ito) [reply version2.en.md]: @yowcow 
fixed🙏  https://github.com/voyagegroup/fluct-rtb-spec/pull/47/commits/d84595c472abaad089d98b8d5808b60788a117e8

**Review by yowcow — APPROVED** [review]
LGTM 😃

**Pull request merged**